# TOPIC'S NAME
IMAGE_TOPIC: "/camera/color/image_raw"
ALTIMETER_TOPIC: "/sf11"
PATCH_TOPIC: "track/path"
POSE_TOPIC: "track/pose"
ODOM_TOPIC: "track/odom"
ORB_IMAGE_TOPIC: "camera/PLK_image"

MISSION_MANAGER_SERVICE: "/mission_manager_camera"
ALARM_SERVICE: "/sensor_alarm"

# ORB PARAMETERS
NUMBER_FEATURES: 20
SCALE_FACTOR: 1.2
N_LEVELS: 8
EDGE_THRESHOLD: 20
FIRST_LEVEL: 0
WTA_K: 2
SCORE_TYPE: 1 # HARRIS_SCORE = 0 | FAST_SCORE = 1
PATCH_SIZE: EDGE_THRESHOLD
FAST_THRESHOLD: 20 

# INVERTED MATRIX OF INTRINSIC PARAMETERS 
INTRINSIC_PARAMETERS: [0.0011, 0, -0.7078,
                      0, 0.0011, -0.4118,
                      0,      0,    1.00]

# TRANSFORMATION PARAMETERS
SCALE: 1.8
HIGH: 0

# SPATIAL ROTATION
ROT_XY: -75
ROT_XZ: 0

# FRAMES ID
ODOM_FRAME_ID: "base_link_init"
PATH_FRAME_ID: "base_link_init"

# DEBUG MODE
DEBUG: True